src/extension.ts

```diff
@@ -231,7 +231,10 @@ export function activate(context: ExtensionContext): any {
   /**
    * Check that D2 is available up front
    */
-  util.checkForD2Install();
+  const checkForD2 = ws.get("checkForInstallAtStart");
```

Comment:
```suggestion
  const checkForD2 = ws.get("checkForInstallAtStart", true);
```

Reply:
I was implicitly correct :), just not explicitly correct.